### PR TITLE
German translation for v13 and v14.

### PR DESCRIPTION
**Pull request by icke-siegen**
I hope I don't break too much :)

### CODE REVIEW

**Comment by Farrser** [comment]
@icke-siegen could you please perform a merge of the upstream `feature/translations` branch now I have updated it

**Comment by icke-siegen** [comment]
Hi, you mean merging the origin into upstream?

S.

Holen Sie sich Outlook für Android<https://aka.ms/ghei36>

________________________________

**Comment by icke-siegen** [comment]
> @icke-siegen could you please perform a merge of the upstream `feature/translations` branch now I have updated it

This is what I did now - seems to find no conflicts:
`
D:\Avolites_Manuals\TitanManual>git status
On branch feature/translations
Your branch is up to date with 'origin/feature/translations'.

nothing to commit, working tree clean

D:\Avolites_Manuals\TitanManual>git merge upstream/feature/translations
Already up to date.

D:\Avolites_Manuals\TitanManual>git status
On branch feature/translations
Your branch is up to date with 'origin/feature/translations'.

nothing to commit, working tree clean

D:\Avolites_Manuals\TitanManual>
`

What else can I do to help?

**Comment by Farrser** [comment]
Ah I think I was able to fix it on my side, I didn't think I'd have permission

**Comment by icke-siegen** [comment]
But I don't have permissions I believe

File: private_sb/git_changes.txt [hunk 1]
```diff
@@ -0,0 +1,1176 @@
+Microsoft Windows [Version 10.0.18363.1139]
```
Comment (Farrser) [inline private_sb/git_changes.txt:1]: Looks like all the files in this folder can be removed for PR

**Review by Farrser — CHANGES_REQUESTED** [review]

**Comment by Farrser** [comment]
I think the best route with this would be to fix these few issues, then I can accept the PR in to the `feature/translations` working branch and we can see where that gets us

File: website/siteConfig.js [hunk 2]
```diff
@@ -28,8 +28,8 @@ const siteConfig = {
   // e.g., for the https://JoelMarcey.github.io site, it would be set like...
   //   organizationName: 'JoelMarcey'
 
-  gaTrackingId: 'GTM-TZMDJV5',
-  gaGtag: true,
+  // gaTrackingId: 'GTM-TZMDJV5',
```
[the remark this answers is not in the document]
Comment (icke-siegen) [reply website/siteConfig.js:31]: I commented it out as Google Analytics almost warrants legal issues in Germany if not properly declared. Put it in again now for the PR.

File: private_sb/git_changes.txt (hunk 1 above)
Comment (icke-siegen) [reply private_sb/git_changes.txt:1]: removed it for the PR (branched it out though)

File: website/pages/de/index.js [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+          <div className="wrapper homeWrapper">{props.children}</div>
+        </div>
+      </div>
+    );
+
+    const ProjectTitle = () => (
+      <h2 className="projectTitle">
+        {siteConfig.title}
+        <small>Deutsches Handbuch für Avolites Titan-Pulte</small>
+		<small>Dies ist eine momentane Arbeitsversion, die Stück für Stück übersetzt wird.<br />Das englische Original-Handbuch gibt es auf <b><a href="https://manual.avolites.com">manual.avolites.com</a></b></small>
```
[the remark this answers is not in the document]
Comment (icke-siegen) [reply website/pages/de/index.js:40]: It was for making the preliminary version available at manuals.avolites.de. Changed this now for the commit/PR - but left translations in for some texts

**Comment by icke-siegen** [comment]
Have committed the changes and pushed into feature/translations in my repo. Do I need to open a new PR now?

**Comment by Farrser** [comment]
No need to open a new PR, this one has been updated. I'll merge this into the `feature/translations` branch of the main repo and review where we're at with it

**Review by Farrser — APPROVED** [review]

**Pull request merged**